app/my-bets/page.tsx

```diff
@@ -23,6 +29,19 @@ export default function MyBetsPage() {
     enabled: !!address,
   });
 
+  const userPositions = data?.userPositions ?? [];
+  const emptyData = data && data.userPositions.length == 0;
+
+  const filterActiveBets = userPositions?.filter(
+    position => position.position.conditions[0].resolved === false
+  );
+
+  const filterClompleteBets = userPositions?.filter(
+    position => position.position.conditions[0].resolved
+  );
```

Comment:
Doesn't userPositions always has a value? No need to conditional check if it is always set with a value.